library/ecp.c

```diff
@@ -1091,10 +1095,37 @@ int mbedtls_ecp_tls_write_point( const mbedtls_ecp_group *grp, const mbedtls_ecp
     buf[0] = (unsigned char) *olen;
     ++*olen;
 
-#endif /* MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL */
+    return( 0 );
+}
+
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) && defined(MBEDTLS_ECDH_C)
+int mbedtls_ecp_tls_13_write_point( const mbedtls_ecp_group *grp, const mbedtls_ecp_point *pt,
+                         int format, size_t *olen,
```

Comment:
From https://tools.ietf.org/html/rfc8446#section-4.2.8.2 I understand that there's no point format negotiation anymore, and the uncompressed form is used throughout. If that understanding is correct, this function should be adjusted accordingly.

Reply:
Hmm... it's not excluded that future groups will use compressed form, though, so perhaps it's better to keep the flexibility for now.